nh_eobs_mobile/static/dev/coffee/src/nhmobilepatient.coffee

```diff
@@ -129,6 +134,10 @@ class NHMobilePatient extends NHMobile
         chartFunc(self, serverData)
       if validTable
         tableFunc(self, serverData)
+      if activeTab is "#graph-content"
+        tableEl.style.display = "none"
+      else
+        graphContent.style.display = "none"
```

Comment:
Does this mean that both table and graph are created in the DOM on page load and just displayed / hidden? Could it impact performance instantiating the table objects when we have 100s of observations even though they don't need to be shown yet?